src/controller/scan/base_controller.go

```diff
@@ -559,9 +559,10 @@ func (bc *basicController) GetReport(ctx context.Context, artifact *ar.Artifact,
 	groupReports := make([][]*scan.Report, len(artifacts))
 
 	var wg sync.WaitGroup
+	var artifactIDs []int64
```

Comment:
Looks like this statement doesn't have any effect.  Remove it?

Reply:
Updated